littleblog/entry/views.py

```diff
@@ -20,37 +23,17 @@ class EntryViewSet(viewsets.ModelViewSet):
     queryset = Entry.objects.all()
     permission_classes = [IsAuthenticated, IsOwnerOrReadOnly]
 
+    @action(detail=True, methods=['patch'])
+    def rate(self, request, pk=None):
+        instance = self.get_object()
```

Comment:
I'd suggest naming this var `entry` - it'd be much easier to read this code. `instance` is too broad, can be anything.

Reply:
Fixed.